### DISCUSSION TITLE
通过转发规则来让词典笔更新一个我们自己的镜像

### DISCUSSION THREAD

**Discussion by 3588044667HZ** [PenUniverse/PenMods#202] [奇技淫巧]
电脑开启热点，设置转发规则，让检查更新的请求返回一个我们编译好的镜像，这样就能绕过adb的密码了。（可以在镜像里加一个ssh或者什么的）
或者在路由器里设置，具体可不可行我没有试，因为fiddler要设置代理，词典笔不支持。其他抓包工具我不了解，仅提供思路

**Comment by Ikaleio** [comment]
但是 s6 的全量包没法解包
主控换了

**Reply by 3588044667HZ** [reply]
不过，我们可以用它的一个老版本的包来试试，让它通过这个办法降级，如果能够成功降级，那么说明它内部大概率是除了md5之外不做其他验证的，我们可以放心做自己的镜像，但是如果不能成功，那么说明它内部是做其他校验的。
  我这个思路可以试一试，我这里有一个s6的老版本在线升级包，大约1.3G，你如果有需要我可以发你。
  ps:颇有些游戏机自制系统的味儿了。

**Reply by Ikaleio** [reply]
binwalk 解包只看见 md5 验证，问题在于没法打包回去了

**Reply by 3588044667HZ** [reply]
能不能参考其他主控的包来设计更新镜像，毕竟系统都差不多
rk主控的包可以解

**Reply by SkySight-666** [reply]
> 不过，我们可以用它的一个老版本的包来试试，让它通过这个办法降级，如果能够成功降级，那么说明它内部大概率是除了md5之外不做其他验证的，我们可以放心做自己的镜像，但是如果不能成功，那么说明它内部是做其他校验的。 我这个思路可以试一试，我这里有一个s6的老版本在线升级包，大约1.3G，你如果有需要我可以发你。 ps:颇有些游戏机自制系统的味儿了。

是普通的OTA增量包吗 如果是的话可以提供一下吗 谢谢（感觉如果带ab更新的话有戏）

**Reply by wyxdlz54188** [reply]
> binwalk 解包只看见 md5 验证，问题在于没法打包回去了

我解出A6pro的包了不知道怎么打包可以帮忙吗
包大小只有500多mb可能是增量包

**Comment by 3588044667HZ** [comment]
> binwalk 解包只看见 md5 验证，问题在于没法打包回去了
